aips/passkeys.md

```diff
@@ -0,0 +1,504 @@
+
+Assuming everything else is correct, verification passes, and other validators agree on the result, this transaction will succeed and be added to the blockchain state.
+
+### Transaction Submission Summary
+
+To summarize, in the Aptos blockchain implementation of the WebAuthn specification, the high-level protocol for authentication assertion is as follows: 
+
+1. The client provides a `challenge` in the form of the `SHA3-256` of the `signing_message` of the `RawTransaction`. We use the `SHA3-256` of the `RawTransaction` instead of the `RawTransaction` to limit the size of the `challenge`, which is part of the `clientDataJSON`.
+2. If the user provides consent, the user agent (browser) will request the authenticator to use a Passkey credential to generate an assertion signature over a message `m` where `m` includes the challenge and returns an authenticator response to the client. 
+3. The client will then send a `SignedTransaction` to the blockchain, where the signature field of the `SignedTransaction` includes the relevant fields in the authenticator response 
+4. The blockchain will verify that the `SignedTransaction`'s signature is over the same message `m` (which includes the `RawTransaction`) and verify that the assertion signature is valid with the corresponding public key
+
+## Reference Implementation
+
+The implementation for the work in this AIP can be found in the [Passkeys Authenticator PR](https://github.com/aptos-labs/aptos-core/pull/10755).
+
+## Testing (Optional)
+
+Testing can be found in the reference implementation provided in the PR link above
+
+## Risks and Drawbacks
+
+### Backup Eligibility 
+
+As mentioned in AIP 61:
+
+> **Passkeys are not always backed up to the cloud** on some platforms (e.g., Microsoft Windows). Furthermore, passkeys introduce a **cross-device problem**: if a user creates a blockchain account on their Apple phone, their passkey secret key is backed up to iCloud, which will not be accessible from that user’s other Android device, Linux device or Windows device, since support for cross-platform backup of passkeys is not yet implemented.
+
+For this reason, it is important to check the `backupEligible` and `backupState` flags from the `authenticatorData` in the registration response to ensure that the passkey has been backed up properly.
+
+#### Operating Systems
+
+See the [operating system reference](https://passkeys.dev/docs/reference/) to better understand what operating systems support passkeys and their limitations. Most notably, passkeys registered on Windows devices will NOT be backup-able as of February 2023. If the user would like the passkey to be backed up, it is recommended that they use a device that allows for multi-device credentials and supports passkey backup.
+
+#### Browsers
+
+See this [device support matrix](https://passkeys.dev/device-support/) for more info on which browsers and operating systems enable backup-able passkeys.
+
+### Wallet liveness
+
```

Comment:
The discussion here is once again not very helpful, you need to talk about impact on dapps, wallets, and not relying parties.